frontend/src/queries/schema/schema-general.ts

```diff
@@ -2327,7 +2327,10 @@ export interface DateRange {
     explicitDate?: boolean | null
 }
 
-export type MultipleBreakdownType = Extract<BreakdownType, 'person' | 'event' | 'group' | 'session' | 'hogql'>
```

Comment:
This needs to be adapted here as well:
https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/insights/filters/BreakdownFilter/taxonomicBreakdownFilterUtils.ts#L14-L17

Reply:
Thanks! Updated in 6b180b9373